components/application/logout.jsx

```diff
@@ -13,7 +13,7 @@ const Logout = ({ store }) => {
   const name = store.user.email
 
   return (
-    <AppBar.Item className={styles.container}>
+    <AppBar.Item>
```

Comment:
Why do you change this? Just curious 🤔

Reply:
`container` class is not defined in logout.css. So I just simply removed it.